includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -886,25 +886,32 @@ public static function orders( $data = null ) {
 			? (float) $meta['_edd_payment_tax_rate'][0]
 			: 0.00;
 
-
 		if ( ! empty( $tax_rate ) ) {
 			// Tax rate is no longer stored in meta.
-			edd_add_order_adjustment( array(
-				'object_id'   => $order_id,
-				'object_type' => 'order',
-				'type_id'     => 0,
-				'type'        => 'tax_rate',
-				'total'       => $tax_rate,
-			) );
+			edd_add_order_adjustment(
+				array(
+					'object_id'     => $order_id,
+					'object_type'   => 'order',
+					'type_id'       => 0,
+					'type'          => 'tax_rate',
+					'total'         => $tax_rate,
+					'date_created'  => $date_created_gmt,
```

Comment:
Should this be `$data->post_modified_gmt`? If not, does it deserve an inline comment because it's different?

Reply:
What I gathered is that the _order adjustments_ used `date_created` and the _refund adjustments_ used `post_modified`. (Though Robin can confirm if that was the goal.)

Reply:
I think my goal was to match it to the order dates, which are set here:

https://github.com/easydigitaldownloads/easy-digital-downloads/blob/ecb6b176ff67914eeebbd6fdd48a69401db7f1d1/includes/admin/upgrades/v3/class-data-migrator.php#L495-L496

This is different than what we ended up doing for customer/order addresses, which use the customer/order date for the creation date, but the migration date for the modified date ([ref](https://github.com/easydigitaldownloads/easy-digital-downloads/pull/7867#issuecomment-628248098)).

And yes, the post_modified is what's used for the refund adjustments to be consistent with refund dates, which are a best guess ([ref](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/upgrades/v3/class-data-migrator.php#L538-L540)).

Mainly, the migrated order adjustments are ideally tracking with the migrated order data so that all are consistent.